firebase-firestore/src/main/java/com/google/firebase/firestore/FieldValue.java

```diff
@@ -182,4 +182,15 @@ public static FieldValue increment(long l) {
   public static FieldValue increment(double l) {
     return new NumericIncrementFieldValue(l);
   }
+
+  /**
+   * Creates a new {@link VectorValue} constructed with a copy of the given array of doubles.
```

Comment:
If you're changing Creates -> Represents per egilmore comment on the js sdk version, apply same changes here AND in the Apple plat/iOS version.

Reply:
Otherwise, LGTM.